README.rst

```diff
@@ -62,33 +62,103 @@ It is suggested to install a pre-commit hook in order to apply Black before push
     $ pre-commit install
 
 
-Project Progres
+Project Progress
 ----------------
 
 View the project Kanban board `here <https://github.com/debrief/pepys-import/projects/3>`_
 
 Setup
 -----
 
-To prepare for running ensure these tools and packages are installed:
+To prepare for running ensure Python 3.6 or later are installed in your system.
+You can check your Python 3 version with the following command::
 
-* Python 3 - at least Python 3.6.4 or later
-* SQL Alchemy 1.3
+    $ python3 --version
 
-For Postgres support and unit tests these packages are also required:
+If you don't have Python 3.6+ in your system, please download it from `python.org <https://www.python.org/downloads/>`_
 
-* geoalchemy2
-* psycopg2
-* nose2 (0.9.1)
+Pip is also necessary to create virtual environment. If you don't have it in your system, please download it::
 
-Dependencies
-------------
+    $ sudo apt-get install python3-pip
+    $ pip3 install virtualenv
```

Comment:
@BarisSari - shouldn't we include `virtualenv` in `requirements.txt` ?

Reply:
For Python 3 you don't need to install virtualenv - it comes with Python itself. You should just be able to do `python -m venv` instead of running `virtualenv`. So to create an environment called `env` just run `python -m venv env`. (I assume we don't need to support Python 2?)

Reply:
We are running `requirements.txt` inside of a virtual environment. So, we should install `virtualenv`, create an environment and activate it first. Therefore, it is not included in requirements.

Reply:
> For Python 3 you don't need to install virtualenv - it comes with Python itself. You should just be able to do `python -m venv` instead of running `virtualenv`. So to create an environment called `env` just run `python -m venv env`. (I assume we don't need to support Python 2?)

I added this command (`python3.6 -m venv venv`) as well but I didn't know that it comes with Python itself.

Reply:
> we don't need to support Python 2
Correct. So, I guess we can remove the step to install virtual.

> not included in requirements
ooh, that's clever :-)

Reply:
Removed:
https://github.com/debrief/pepys-import/pull/66/commits/a9f09e17530f0fb271bb3ecaf72272b56cb07bd4